src/pages/home/report/EmojiPickerMenu/index.js

```diff
@@ -110,7 +105,9 @@ class EmojiPickerMenu extends Component {
                     this.props.onEmojiSelected(this.state.filteredEmojis[this.state.highlightedIndex].code);
                 }
             };
-            document.addEventListener('keydown', this.keyDownHandler);
+
+            // Keyboard events are not bubbling in RN-Web
```

Comment:
This comment isn't super helpful in it's current state – if you include it I think it's better to explain _why_ bubbling in this case is necessary for this feature to work.